apstools/devices.py

```diff
@@ -897,6 +897,25 @@ def state(self):
         return result
 
 
+class TrackingSignal(Signal):
+    """
+    Signal to decide if undulator will be tracked while changing the
+    monochromator energy.
+    """
+
+    def check_value(self, value):
+        """
+        Check if the value is a boolean.
+
+        Raises
+        ------
+        ValueError
+        """
+        if type(value) != bool:
+            msg = 'tracking is boolean, it can only be True or False.'
```

Comment:
It is more efficient when the message is created within the raised exception.  Only when `raise` is called will the message text be created.  So:

```
raise ValueError("tracking is boolean, it can only be True or False.")
```